regression-test/suites/query_p0/sql_functions/bitmap_functions/test_bitmap_function.groovy

```diff
@@ -833,6 +833,8 @@ suite("test_bitmap_function") {
     //         orthogonal_bitmap_intersect_count(id_bitmap, tag, 0) as count2_bitmap from test_orthog_bitmap_intersect;
     // """
 
+    /////////////////////////////
+    // test bitmap base64
```

Comment:
u should add test cases in `regression-test/suites/nereids_function_p0/scalar_function/B.groovy` too

Reply:
already added case for nereids in `regression-test/suites/query_p0/sql_functions/bitmap_functions/test_bitmap_function.groovy`